src/styleUtils.js

```diff
@@ -64,19 +63,18 @@ define(
 		);
 	};
 	
-	// Uses the Flash based downloadify library to save files to the local file system
+	// Uses the FileSaver.js library to save files to the local file system
 	var saveCsl = function (
 			cslCodeOrData,
 			styleId,
 			comment // optional, if comment is included cslCodeOrData
 			        // must be a CSLEDIT_Data instance
 			) {
 		var dialog = $('<div title="Save CSL Style">' +
-				'<div id="downloadify" style="padding-left: 300px"></div>' +
-				'<div id="installFlash" style="padding-left:50px"></div>' +
+				'<div id="download" style="padding-left: 300px"></div>' +
 				'<div id="refManagerInstructions"><\/div>' +
 				'<\/div>'),
```

Comment:
Yes, thanks. That looks relevant. Had some trouble finding relevant documentation myself.

Based on that, I see no reason why the slashes need to be escaped.